alicloud/resource_alicloud_cs_kubernetes.go

```diff
@@ -140,16 +140,30 @@ func resourceAlicloudCSKubernetes() *schema.Resource {
 				Optional:         true,
 				ForceNew:         true,
 				Sensitive:        true,
-				ConflictsWith:    []string{"key_name"},
+				ConflictsWith:    []string{"key_name", "kms_encrypted_password"},
 				DiffSuppressFunc: csForceUpdateSuppressFunc,
 			},
 			"key_name": {
 				Type:             schema.TypeString,
 				ForceNew:         true,
 				Optional:         true,
-				ConflictsWith:    []string{"password"},
+				ConflictsWith:    []string{"password", "kms_encrypted_password"},
 				DiffSuppressFunc: csForceUpdateSuppressFunc,
 			},
+			"kms_encrypted_password": {
+				Type:          schema.TypeString,
+				ForceNew:      true,
+				Optional:      true,
+				ConflictsWith: []string{"password", "key_name"},
+			},
+			"kms_encryption_context": {
+				Type:     schema.TypeMap,
+				Optional: true,
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					return d.Get("kms_encrypted_password") == ""
```

Comment:
`return d.Get("kms_encrypted_password") == ""`
to `return d.Get("kms_encrypted_password").(string) == ""`